content/docs/use-cases/ci-cd-for-machine-learning.md

```diff
@@ -0,0 +1,69 @@
+# Continuous Integration and Deployment for Machine Learning
+
+MLOps and DataOps apply DevOps methodologies to machine learning (ML) and data
+management. The fields are rapidly evolving beyond a mere set of best practices,
+and now encompass the entire product lifecycle management. A core component is
+the automation of testing and deployment: CI/CD (continuous integration and
+continuous delivery).
```

Comment:
delivery or deployment? :)

Reply:
I prefer *deployment* but think the original acronym uses *delivery*.
My compromise was to use *deployment* in the preceding sentence...

Reply:
> delivery or deploymentI'd stick to only one of these terms.

Reply:
> delivery or deployment

I'd stick to only one of these terms. Deployment seems more appropriate here IMO.